pkgs/applications/networking/instant-messengers/beeper/default.nix

```
@@ -0,0 +1,59 @@
+{ lib, fetchurl, mkDerivation, appimageTools, libsecret, makeWrapper }:
+let
+  pname = "beeper";
+  version = "3.62.20";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://download.todesktop.com/2003241lzgn20jd/${name}.AppImage";
```

Please avoid using this.

This creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

```suggestion
    url = "https://download.todesktop.com/2003241lzgn20jd/beeper-${version}.AppImage";
```